.travis.yml

```diff
@@ -20,7 +19,9 @@ script:
   - stack setup --no-terminal
   - stack build --ghc-options=-Werror --no-terminal
   - stack test --ghc-options=-Werror --no-terminal
-  - stack haddock --no-terminal
```

Comment:
rather add a `stack.yaml` for nightly and `if [ $STACK_YAML = "stack-nightly.yaml" ]; ...`

Reply:
Agreed.

Reply:
@phadej @jkarni Currently `stack haddock` does not work unless it's using the nightly. Do you mean to only run `stack haddock` for `stack-nightly.yaml`?

Reply:
yes

Reply:
I noticed that the `stack-ghc-8.2.1.yaml` was already using a nightly so my latest commit just runs `stack haddock` for that.